mappings/net/minecraft/reference/Achievements.mapping

```diff
@@ -12,6 +12,10 @@ CLASS none/np net/minecraft/reference/Achievements
 	FIELD K FULL_BEACON Lnone/no;
 	FIELD L EXPLORE_ALL_BIOMES Lnone/no;
 	FIELD M OVERPOWERED Lnone/no;
+	FIELD a lowestY I
```

Comment:
These values adjust every time a new achievement is added. So if the past lowestY was -25 and you put one at -300, the lowestY would become -300. min/max implies (to me) that it is a limitation, however this is just keeping track of the highest and lowest coordinates.